Amplify/DevMenu/View/InfoRow.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// View corresponding to each row in Device Information Screen / Environment Information Screen
+@available(iOS 13.0.0, *)
+struct InfoRow: View {
+    private let notAvailable = "Not Available"
+    var key: String
+    var value: String
+
+    init(item: DeviceInfoItem) {
+        self.key = item.type.key
+        if item.type.value.isEmpty {
+            self.value = notAvailable
+        } else {
+            self.value = item.type.value
+        }
+    }
+
+    init(item: EnvironmentInfoItem) {
+        self.key = item.key
+        if item.value.isEmpty {
+            self.value = notAvailable
+        } else {
+            self.value = item.value
+        }
+    }
```

Comment:
As mentioned in the previous comment please check if you can make this generic using a protocol.